vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -99,7 +99,7 @@ def test_getitem_unknown_args(self, standard_px_chart):
 
     @pytest.mark.parametrize(
```

Comment:
These test cases should render the same way as before, it's just that the way we apply the values is different now.